chain/chain_config.go

```diff
@@ -259,30 +291,30 @@ func (c *Config) CheckCompatible(newcfg *Config, height uint64) *chain.ConfigCom
 	return lasterr
 }
 
-type forkBlockNumber struct {
-	name        string
-	blockNumber *big.Int
-	optional    bool // if true, the fork may be nil and next fork is still allowed
-}
-
-func (c *Config) forkBlockNumbers() []forkBlockNumber {
```

Comment:
do we need to comment this out or remove?

Reply:
It was used in the  CheckConfigForkOrder function, which is now stopped abruptly because of dynamic forks. But it is still used, so I didn't want to lose the code. We can remove it if you think it will be okay.